ddev/ddev
# feat: install add-on for unknown command if available, fixes #6352

## The Issue

- #6352

## How This PR Solves The Issue

Tries to find a matching official add-on for unknown command and installs it.

## Manual Testing Instructions

Try things from `ddev get --list` in the DDEV project folder:
- `ddev phpmyadmin`
- `ddev redis`
- `ddev adminer`
- `ddev redis-7`

After installing the add-on, repeat the same command, i.e. install the Redis add-on using `ddev redis` and then run `ddev redis` again to check.

## Automated Testing Overview



## Release/Deployment Notes
